warehouse_simulator.py

```diff
@@ -0,0 +1,164 @@
+"""Create simulator from real life. This can be booking room in hotel, visit
+to casino, visit to bar. Create 3-4 objects, that can describe situation.
+Objects should contain attributes and methods to simulate some use cases.
+Completed program should print object states, it actions (methods) and objects
+interaction.
+"""
+
+
+class Warehouse(object):
+    """This is simulator of Warehouse.
+
+    Starting parameters:
+    material_limit - material limit in m^3
+    money - starting sum of money in dollars
+    cheat - difference between purchase price and sale price
+    extend_cost - cost in dollars of Warehouse extending at 1m^3
+    """
+    material_limit = 5000
```

Comment:
do we really need so many class variables?

Reply:
solved.